docs/reference/search/profile.asciidoc

```diff
@@ -602,3 +584,137 @@ just not the physical layout in the response, so it is sufficient to just analyz
 ignore it's children if you find the details too tricky to interpret.
 
 Hopefully this will be fixed in future iterations, but it is a tricky problem to solve and still in-progress :)
+
+=== Profiling Aggregations
+
+==== `aggregations` Section
+
+The `aggregations` section contains detailed timing of the aggregation tree executed by a particular shard.
+The overall structure of this aggregation tree will resemble your original Elasticsearch request.  Let's consider
+the following example aggregations request:
+
+[source,js]
+--------------------------------------------------
+curl -XGET "http://localhost:9200/house-prices/_search" -d'
+{
+  "profile": true,
+  "size": 0,
+  "aggs": {
+    "property_type": {
+      "terms": {
+        "field": "propertyType"
+      },
+      "aggs": {
+        "avg_price": {
+          "avg": {
+            "field": "price"
+          }
+        }
+      }
+    }
+  }
+}'
+--------------------------------------------------
+
+Which yields the following aggregation profile output
+
+[source,js]
+--------------------------------------------------
+"aggregations": [
+  {
+    "type": "org.elasticsearch.search.aggregations.bucket.terms.GlobalOrdinalsStringTermsAggregator",
+    "description": "property_type",
+    "time": "4280.456978ms",
+    "breakdown": {
+      "reduce": 0,
+      "build_aggregation": 49765,
+      "initialise": 52785,
+      "collect": 3155490036
+    },
+    "children": [
+      {
+        "type": "org.elasticsearch.search.aggregations.metrics.avg.AvgAggregator",
+        "description": "avg_price",
+        "time": "1124.864392ms",
+        "breakdown": {
+          "reduce": 0,
+          "build_aggregation": 1394,
+          "initialise": 2883,
+          "collect": 1124860115
+        }
+      }
+    ]
+  }
+]
+--------------------------------------------------
+
+From the profile structure we can see our `property_type` terms aggregation which is internally represented by the
+`GlobalOrdinalsStringTermsAggregator` class and the sub aggregator `avg_price` which is internally represented by the `AvgAggregator` class. The `type` field displays the class used internally to represent the aggregation. The `description` field displays the name of the aggregation.
+
+The `"time"` field shows that it took ~4 seconds for the entire aggregation to execute.  The recorded time is inclusive
```

Comment:
You use `time` and `"time"`.  I'd make it consistent: `time`.  Same for `"breakdown"`